app/controllers/application_controller.rb

```diff
@@ -2125,7 +2125,8 @@ def set_global_session_data
       case controller_name
 
       # These controllers don't use breadcrumbs, see above get method to store URL
-      when "dashboard", "report", "support", "alert", "jobs", "ui_jobs", "miq_ae_tools", "miq_policy", "miq_action", "miq_capacity", "chargeback", "service"
+      # when "dashboard", "report", "support", "alert", "alert_center", "jobs", "ui_jobs",
```

Comment:
Rubocop complains of an empty 'when' block. I commented it out rather than removing it for clarity of what sections are being ignored.